src/IO/S3/Credentials.cpp

```diff
@@ -1,4 +1,9 @@
+#include <exception>
+#include <variant>
 #include <IO/S3/Credentials.h>
+#include <boost/algorithm/string/classification.hpp>
+#include <Poco/Exception.h>
+#include "Common/Exception.h"
```

Comment:
everything except `#include <IO/S3/Credentials.h>` should be removed or added below under `#if USE_AWS_S3`

Reply:
done